### PR TITLE
Editorial edit suggestions for Basic Concepts section

### PR DESCRIPTION
**Pull request by grantnoble**
Again, "verifiable" credentials and presentations. Some grammar and rewording, and some linking.

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/vc-data-model/pull/516.html" title="Last updated on Apr 2, 2019, 10:51 PM UTC (abb4436)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/516/1ad7698...grantnoble:abb4436.html" title="Last updated on Apr 2, 2019, 10:51 PM UTC (abb4436)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1362,17 +1373,17 @@ <h3>Issuance Date</h3>
...
           <dd>
-A <a>verifiable credential</a> MUST have an <code>issuanceDate</code> <a>property</a>.
-The value of the <code>issuanceDate</code> <a>property</a> MUST be a string
-value of an [[!RFC3339]] combined date and time string representing the date
-and time the <a>credential</a> was issued. Note that this date represents the
-earliest date when the information associated with the
+A <a>verifiable credential</a> MUST have an <code>issuanceDate</code>
+<a>property</a>. The value of the <code>issuanceDate</code> <a>property</a> MUST
+be a string value of an [[!RFC3339]] combined date and time string representing
+the date and time the <a>credential</a> was issued. Note that this date
```
Comment (ken-ebert) [inline index.html]: <a>credential</a> should be <a>verifiable credential</a>

Comment (ken-ebert) [reply index.html]: the rest of the changes look good to me.

Comment (grantnoble) [reply index.html]: Fixed.

File: index.html [hunk 2]
```diff
@@ -1181,7 +1189,7 @@ <h3>Types</h3>
 <a href="#status">credentialStatus</a>&nbsp;object
               </td>
               <td>
-A valid <a>credential</a> status <a>type</a>. For example,<br>
+A valid <a>verifiable credential</a> status <a>type</a>. For example,<br>
```
Comment (msporny) [inline index.html]: hmm, you can use status w/ just plain credentials... so this is wrong, we meant credential.

File: index.html [hunk 3]
```diff
@@ -1210,25 +1218,25 @@ <h3>Types</h3>
         </table>
 
         <p>
-All <a>credentials</a>, <a>presentations</a>, and encapsulated objects MUST
-specify, or be associated with, additional more narrow <a>types</a> (like
-<code>UniversityDegreeCredential</code>, for example) so software systems can
-process this additional information.
+All <a>verifiable credentials</a>, <a>verifiable presentations</a>, and
```
Comment (msporny) [inline index.html]: We meant credentials and presentations, specifically did not mean verifiable in this case.

File: index.html [hunk 4]
```diff
@@ -1210,25 +1218,25 @@ <h3>Types</h3>
...
         <p>
 When processing encapsulated objects defined in this specification, (for
 example, objects associated with the <code>credentialSubject</code> object
 or deeply nested therein), software systems SHOULD use the <a>type</a>
 information specified in encapsulating objects higher in the hierarchy.
-Specifically, an encapsulating object, such as <a>credential</a>, SHOULD
-convey the associated object <a>types</a> so that <a>verifiers</a> can quickly
-determine the contents of an associated object based on the encapsulating
-object <a>type</a>.
+Specifically, an encapsulating object, such as a <a>verifiable credential</a>,
```
Comment (msporny) [inline index.html]: This change is wrong... we meant "credential"

File: index.html [hunk 5]
```diff
@@ -1210,25 +1218,25 @@ <h3>Types</h3>
...
+SHOULD convey the associated object <a>types</a> so that <a>verifiers</a> can
+quickly determine the contents of an associated object based on the
+encapsulating object <a>type</a>.
         </p>
 
         <p>
-For example, a <a>credential</a> object with the <code>type</code> of
+For example, a <a>verifiable credential</a> object with the <code>type</code> of
```
Comment (msporny) [inline index.html]: This is not right, we meant credential.

File: index.html [hunk 6]
```diff
@@ -1331,7 +1342,7 @@ <h3>Issuer</h3>
 RECOMMENDED that dereferencing the <a>URI</a>, or the <code>id</code> property,
 results in a document containing machine-readable information about the
 <a>issuer</a> that can be used to <a>verify</a> the information expressed in
-the <a>credential</a>.
+the <a>verifiable credential</a>.
```
Comment (msporny) [inline index.html]: This change is not correct, we meant "credential".

File: index.html [hunk 7]
```diff
@@ -1362,17 +1373,17 @@ <h3>Issuance Date</h3>
 
         <p>
 This specification defines the following <a>property</a> for expressing the
-date and time when the <a>credential</a> was issued:
+date and time when the <a>verifiable credential</a> was issued:
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 8]
```diff
@@ -1362,17 +1373,17 @@ <h3>Issuance Date</h3>
...
           <dd>
-A <a>verifiable credential</a> MUST have an <code>issuanceDate</code> <a>property</a>.
-The value of the <code>issuanceDate</code> <a>property</a> MUST be a string
-value of an [[!RFC3339]] combined date and time string representing the date
-and time the <a>credential</a> was issued. Note that this date represents the
-earliest date when the information associated with the
+A <a>verifiable credential</a> MUST have an <code>issuanceDate</code>
+<a>property</a>. The value of the <code>issuanceDate</code> <a>property</a> MUST
+be a string value of an [[!RFC3339]] combined date and time string representing
+the date and time the <a>verifiable credential</a> was issued. Note that this
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 9]
```diff
@@ -1475,16 +1486,16 @@ <h3>Proofs (Signatures)</h3>
         <h3>Expiration</h3>
 
         <p>
-This specification defines the following <a>property</a> for the
-expression of <a>credential</a> expiration information:
+This specification defines the following <a>property</a> for the expression of
+<a>verifiable credential</a> expiration information:
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 10]
```diff
@@ -1475,16 +1486,16 @@ <h3>Proofs (Signatures)</h3>
...
 
         <dl>
           <dt><var>expirationDate</var></dt>
           <dd>
-The value of the <code>expirationDate</code> <a>property</a> MUST
-be a string value of an [[!RFC3339]] combined date and time string representing
-the date and time the <a>credential</a> ceases to be valid.
+The value of the <code>expirationDate</code> <a>property</a> MUST be a string
+value of an [[!RFC3339]] combined date and time string representing the date and
+time the <a>verifiable credential</a> ceases to be valid.
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 11]
```diff
@@ -1544,10 +1554,10 @@ <h3>Status</h3>
         </dl>
 
         <p>
-The precise contents of the <a>credential</a> status information is determined
-by the specific <code>credentialStatus</code> <a>type</a> definition, and
-varies depending on factors such as whether it is simple to implement or if it
-is privacy-enhancing.
+The precise contents of the <a>verifiable credential</a> status information is
```
Comment (msporny) [inline index.html]: nope, credential.

Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 10 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 9 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 8 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 7 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 6 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 5 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 4 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (grantnoble) [reply index.html]: Fixed.

**Comment by grantnoble** [comment]
Merge conflicts resolved.

**Comment by msporny** [comment]
There are now more merge conflicts. :(

File: index.html [hunk 12]
```diff
@@ -1000,18 +1003,18 @@ <h3>Contexts</h3>
...
         </p>
 
         <p class="note">
-This document uses the examples context URI
-(<code>https://www.w3.org/2018/credentials/examples/v1</code>) for the purposes of
-demonstrating examples. The examples context URI must not be used for any
-other purpose, such as in pilot or production systems.
+This document uses the examples context <a>URI</a>
+(<code>https://www.w3.org/2018/credentials/examples/v1</code>) for the purposes
+of demonstrating examples. The examples context <a>URI</a> MUST NOT be used for
```
Comment (msporny) [inline index.html]: This transforms a non-normative "must not" to a normative "MUST NOT" and would kick us into another Candidate Recommendation phase. Keep the non-normative nature of this statement, please.

File: index.html [hunk 13]
```diff
@@ -1389,12 +1399,12 @@ <h3>Issuance Date</h3>
         <dl>
           <dt><var>issuanceDate</var></dt>
           <dd>
-A <a>verifiable credential</a> MUST have an <code>issuanceDate</code> <a>property</a>.
-The value of the <code>issuanceDate</code> <a>property</a> MUST be a string
-value of an [[!RFC3339]] combined date and time string representing the date
-and time the <a>credential</a> was issued. Note that this date represents the
-earliest date when the information associated with the
-<code>credentialSubject</code> <a>property</a> became valid.
+A <a>credential</a> MUST have an <code>issuanceDate</code> <a>property</a>. The
```
Comment (msporny) [inline index.html]: This is a non-substantive change as the group meant this property to apply to both "credentials" and "verifiable credentials".

Comment (msporny) [reply index.html]: @grantnoble I'm just noting here why this change is ok. You typically don't want to change normative statements (anything w/ MUST, SHOULD, MAY) during the Candidate Recommendation phase.

**Comment by grantnoble** [comment]
Merge conflicts resolved

File: index.html (hunk 12 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 13 above)
Comment (grantnoble) [reply index.html]: Thanks!

**Pull request merged**